pep-0299.txt

```diff
@@ -5,116 +5,125 @@ Last-Modified: $Date$
-    function, and because arguments are taken from sys.argv it is not
-    possible to pass specific arguments without changing the argument
-    list seen by all other modules.  (Imagine a threaded Python
-    program, with two threads wishing to invoke the standalone
-    functionality of different modules with different argument lists)
+is sometimes seen, but there exists no standard name for the function,
+and because arguments are taken from sys.argv it is not possible to
+pass specific arguments without changing the argument list seen by all
+other modules.  (Imagine a threaded Python program, with two threads
+wishing to invoke the standalone functionality of different modules
+with different argument lists)
 
 
 Proposal
+========
 
-    The standard name of the 'main function' should be '__main__'.
-    When a module is invoked on the command line, such as
+The standard name of the 'main function' should be `__main__`. When a
+module is invoked on the command line, such as::
 
-        python mymodule.py
+    python mymodule.py
 
-    then the module behaves as though the following lines existed at
-    the end of the module (except that the attribute __sys may not be
-    used or assumed to exist elsewhere in the script):
+then the module behaves as though the following lines existed at the
+end of the module (except that the attribute __sys may not be used or
+assumed to exist elsewhere in the script)::
 
-        if globals().has_key("__main__"):
-            import sys as __sys
-            __sys.exit(__main__(__sys.argv))
+    if globals().has_key("__main__"):
+        import sys as __sys
+        __sys.exit(__main__(__sys.argv))
 
-    Other modules may execute
+Other modules may execute::
 
-        import mymodule
-        mymodule.__main__(['mymodule', ...])
+    import mymodule mymodule.__main__(['mymodule', ...])
 
-    It is up to mymodule to document thread-safety issues or other
-    issues which might restrict use of __main__.  (Other issues might
-    include use of mutually exclusive GUI modules, non-sharable
-    resources like hardware devices, reassignment of sys.stdin/stdout,
-    etc)
+It is up to ``mymodule`` to document thread-safety issues or other
+issues which might restrict use of `__main__`.  (Other issues might
+include use of mutually exclusive GUI modules, non-sharable resources
+like hardware devices, reassignment of ``sys.stdin``/``stdout``, etc)
 
 
 Implementation
+==============
 
-    In modules/main.c, the block near line 385 (after the
-    PyRun_AnyFileExFlags call) will be changed so that the above code
-    (or its C equivalent) is executed.
+In ``modules/main.c``, the block near line 385 (after the
+``PyRun_AnyFileExFlags`` call) will be changed so that the above code
+(or its C equivalent) is executed.
 
 
 Open Issues
+===========
 
-    - Should the return value from __main__ be treated as the exit value?
+Should the return value from``__main__`` be treated as the exit
```

Comment:
> (notice indentation in the second line)

I was unsure if I should use a list of a definition list which looks more like Question/Answer, but sure.